### PR TITLE
remove existing hydra references and replace with samvera and rework footer

### LINKED ISSUE #1419: Change Hydra references in Scholar to Samvera
Code doesn't need to be changed, but the following files have references to Hydra that need to be changed to Samvera:

https://github.com/uclibs/scholar_uc/blob/develop/config/locales/sufia.en.yml

https://github.com/uclibs/scholar_uc/blob/develop/app/views/static/_about_text.html.erb

https://github.com/uclibs/scholar_uc/blob/develop/README.md

https://github.com/uclibs/scholar_uc/blob/develop/spec/views/shared/_footer.html.erb_spec.rb

https://github.com/uclibs/scholar_uc/blob/develop/app/views/layouts/scholar/_partner_branding.html.erb

### PR DESCRIPTION
**Pull request by lawhorkl**
closes #1419 and #1549 

changes all files mentioned in #1419 

reworks footer to include proper UC links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Scholar@UC is a digital repository that enables the University of Cincinnati community to share its research and scholarly work with a worldwide audience. Faculty and staff can use Scholar@UC to collect their work in one location and create a durable and citeable record of their papers, presentations, publications, datasets, or other scholarly creations.
 
-Scholar@UC uses Project Hydra's [Hyrax gem](https://github.com/projecthydra-labs/hyrax). The source for UC's Hyrax-based app can be found at https://github.com/uclibs/scholar_uc
+Scholar@UC uses Project Samvera's [Hyrax gem](https://github.com/projecthydra-labs/hyrax). The source for UC's Hyrax-based app can be found at https://github.com/uclibs/scholar_uc
```
Comment (hortongn) [inline README.md]: @lawhorkl When the name changed from "Project Hydra" they dropped the "Project" part.  So replace "Project Hydra" with just "Samvera" wherever the change is needed.

File: app/views/static/_about_text.html.erb [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
+  <%=t('hyrax.product_name') %> is an open source, agile development project supported in partnership by the <%=t('hyrax.institution_name') %> Libraries and UC Information Technologies. Our source code repository is found on github.com in two places: <a href="https://github.com/uclibs/scholar_uc" target="_blank">our application code</a> and a <a href="https://github.com/projecthydra/hyrax" target="_blank">Ruby on Rails engine</a>.  Our application is built on top of <a href="http://projecthydra.org" target="_blank">Project Samvera</a>, a repository framework used by institutions worldwide to provide access to their digital content, and an open source community to facilitate shared software development.
```
Comment (hortongn) [inline app/views/static/_about_text.html.erb]: @lawhorkl Change `http://projecthydra.org` to `https://samvera.org` and change `https://github.com/projecthydra/hyrax` to `https://github.com/samvera/hyrax`

**Comment by hortongn** [comment]
@lawhorkl in the README, let's go ahead and change the instructions for starting Solr to use `samvera-development` instead of `hydra-development`.  That also means editing config/solr_wrapper_test.yml, config/solr.yml.bamboo, and config/solr.yml.sample to use `samvera-` instead of `hydra-`

File: app/views/static/_about_text.html.erb (hunk 2 above)
Comment (hortongn) [inline app/views/static/_about_text.html.erb]: @lawhorkl just use "Samvera" instead of "Project Samvera" here.

**Comment by crowesn** [comment]
@lawhorkl 

New prototype for footer.

footer

**Comment by lawhorkl** [comment]
@hortongn updated the PR according to @crowesn and @newmanld 's feedback, should be ready to go.

**Comment by hortongn** [comment]
@crowesn @newmanld @lawhorkl I think the link in the footer that just says "Alerts" is very vague.  Can we go with "UC Alerts"?

**Comment by crowesn** [comment]
Fine with me. Or just get rid of it.


--

Sean Crowe
Digital Analyst and Developer Librarian

Digital Projects and Repositories.
University of Cincinnati Libraries
PO Box 210033
Cincinnati OH 45221-0033
Tel: (513) 556-1899
Fax: (513) 556-4393
Email: sean.crowe@uc.edu<mailto:sean.crowe@uc.edu>
gchat: crowesn










On Jul 21, 2017, at 10:49 AM, Glen Horton <notifications@github.com<mailto:notifications@github.com>> wrote:


@crowesn<https://github.com/crowesn> @newmanld<https://github.com/newmanld> @lawhorkl<https://github.com/lawhorkl> I think the link in the footer that just says "Alerts" is very vague. Can we go with "UC Alerts"?

**Comment by hortongn** [comment]
Good job!

**Comment by newmanld** [comment]
I am asking that we show all of the links that are in the footer on uc.edu and digital.libraries.uc.edu on the line that begins with ‘Alerts’ and ends with ‘University of Cincinnati’ .  Having a horizontal line devoted solely to these UC links is one way to group them and make that clear, but I’ve acceded to an attempt to come up with something visually more attractive.  I’d agree with the variation of ‘UC Alerts’ but not with getting rid of the link altogether.

**Pull request merged**